src/core.ts

```diff
@@ -99,7 +99,9 @@ export type AirgapAuth =
   /** A user-initiated interaction event that was just dispatched. */
   | UIEvent
   /** A `load` event from a just-loaded airgap.js script element (which implies auth by being loaded before airgap.js) */
-  | Event;
+  | Event
+  /** Consent change authorization key automatically conferred by airgap via setAuth() API for UI modules */
+  | symbol;
```

Comment:
Sorry I haven't been keeping up but what will this key be used for in the end? Also, since this key is automatically made is it just a random key?

Reply:
This key will be used by the UI module while setting consent

Reply:
thanks for explaining!